mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -451,11 +471,14 @@ private class LogisticAggregator(
    * @param label The label for this data point.
    * @param data The features for one data point in dense/sparse vector format to be added
    *             into this aggregator.
+   * @param sampleWeight The weight for over-/undersamples each of training sample.
    * @return This LogisticAggregator object.
    */
-  def add(label: Double, data: Vector): this.type = {
+  def add(label: Double, sampleWeight: Double = 1.0, data: Vector): this.type = {
```

Comment:
No need for a default value for sampleWeight.  Or if you want it for other purposes later on, it should probably be the last argument.